classes/scheduledTask/ScheduledTask.php

```diff
@@ -52,7 +52,8 @@ public function __construct($args = [])
         $fileMgr = new PrivateFileManager();
 
         $scheduledTaskFilesPath = realpath($fileMgr->getBasePath()) . '/' . ScheduledTaskHelper::SCHEDULED_TASK_EXECUTION_LOG_DIR;
-        $this->_executionLogFile = "{$scheduledTaskFilesPath}/" . str_replace(' ', '', $this->getName()) .
+        $classNameParts = explode('\\', get_class($this)); // Separate namespace info from class name
```

Comment:
Not important, but you can use `$this::class`.

Reply:
Thanks a lot @jonasraoni! I will leave it so here, but change it for main...